Ontology/Asset/Component/Valve/Valve.json

```diff
@@ -18,7 +18,8 @@
       },
       "name": "pipeDiameter",
       "schema": "double",
-      "writable": true
+      "writable": true,
+      "comment": "deprecate this property as it has been replaced by pipeInlet and pipeOutlet"
```

Comment:
Is this how we deprecate? Wouldn't it be better to simply remove and to increment DTMI version?

Reply:
Agreed